package.json

```diff
@@ -5,6 +5,7 @@
   "description": "Boilerplate for Create-React-App with an Express.js API",
   "main": "server.js",
   "scripts": {
+    "api": "json-server --watch db.json --port 5000",
```

Comment:
You don't need this script, as you start the json-server in your server.js

Reply:
okay. Will delete :)